test/default-messages.js

```diff
@@ -1,37 +1,32 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath, [ __dirname + '/messages/defaults.js' ]);
```

Comment:
let's use `path.join` here - ideally the `/` is never hardcoded, so things work on Windows.